modules/ocf_kvm/manifests/init.pp

```diff
@@ -51,4 +51,15 @@
       owner     => 'root',
       show_diff => false;
   }
+
+  # firewall input rules, allow all ssh, allow ntp
+  ocf::firewall::firewall46 {
+    '101 accept all ssh':
```

Comment:
Isn't this the same as `ocf::firewall::allow_ssh`? Also I don't see ntp here as described in the above comment.